commands/initServerCommands.go

```diff
@@ -0,0 +1,18 @@
+package commands
+
+import (
+	"github.com/ditrit/verdeter"
+	"github.com/ditrit/verdeter/validators"
+)
+
+func initServerCommands(cfg *verdeter.VerdeterCommand) {
+	cfg.GKey("server.max_timeout", verdeter.IsInt, "", "maximum timeout (in second)")
```

Comment:
change by `server.timeout`, be more explicit on the description and add default in description